staging/src/k8s.io/client-go/tools/cache/thread_safe_store.go

```diff
@@ -283,11 +288,35 @@ func (c *threadSafeMap) Replace(items map[string]interface{}, resourceVersion st
 	c.lock.Lock()
 	defer c.lock.Unlock()
 	c.items = items
+	c.rv = ""
```

Comment:
shouldn't this be using the resourceVersion passed in instead of looking at all the individual RVs of the objects?